File: Modul-4/Soal-2/PRAK402-2410817210008-AhmadUlyani.c
#include <stdio.h>

void hitung_ganjil_genap(int n) {
    for (int i = 1; i <= n; i++) {
        if (i % 2 != 0) { // Jika ganjil
            printf("%d ", i);
        }
    }
    printf("\n");

    for (int i = n; i >= 2; i--) {
        if (i % 2 == 0) { // Jika genap
            printf("%d ", i);
        }
    }
    printf("\n");
}

int main() {
    int n;
    scanf("%d", &n);
    hitung_ganjil_genap(n);

    return 0;
}

File: Modul-4/Soal-2/PRAK402-2410817210008-AhmadUlyani.py
def hitung_ganjil_genap(n):
    ganjil = [i for i in range(1, n + 1) if i % 2 != 0]
    genap = [i for i in range(n, 1, -1) if i % 2 == 0]

    print(" ".join(map(str, ganjil)))
    print(" ".join(map(str, genap)))

n = int(input())
hitung_ganjil_genap(n)